doc/user/content/sql/create-source/postgres.md

```diff
@@ -102,6 +102,8 @@ If you stop or delete Materialize without first dropping the Postgres source, th
   SET STORAGE PLAIN;
   ```
 - Tables replicated into Materialize should not be truncated. If a table is truncated while replicated, the whole source becomes inaccessible and will not produce any data until it is re-created.
+- Since Postgres sources are materialized by default, Postgres table sources must be smaller than the available memory.
+- Records deleted from Postgres do not take up space in memory.
```

Comment:
This one doesn't really fit the bill of "Restrictions on Postgres sources", it's more of a good-to-know.  I can include it in the upcoming CDC guide.

Reply:
Probably not even worth mentioning. All sources have this behaviour (that they don't retain deleted rows). I'm not sure why it was asked